hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -845,6 +845,11 @@
         rely on it. If true, it increases the speed of query processes in the map. It only works when
         <in-memory-format> is set to BINARY and performs a pre-caching on the entries queried. Its default
         value is false.
+        * <preprocessing-policy>
+        Pre-processing policy for this map.
+        Valid values are:
+        CREATION_TIME (default): Objects of supported types are pre-processed when they are created and updated.
```

Comment:
Default is `OFF`. Also, maybe add some link or additional comment as to what kind of pre-processing is being done.
Same in `hazelcast-default.xml`, `hazelcast-config-3.12.xsd` and `hazelcast-fullconfig.xml`.

Reply:
good catch. Thanks